deepchem/data/data_loader.py

```diff
@@ -875,9 +875,88 @@ class provides convenience files to lead FASTA data and
   learning tasks.
   """
 
-  def __init__(self):
-    """Initialize loader."""
-    pass
+  def __init__(self,
+               featurizer=OneHotFeaturizer,
```

Comment:
You're passing in a class rather than a featurizer object here :). I think you want to do something like
```
featurizer=OneHotFeaturizer()
```
to make an object. I think that should work. Also make sure to add a type signature for `featurizer`

Reply:
Ok just noticed below that you initialize `OneHotFeaturizer` with different arguments for different charsets. A better default might be to use
```
featurizer=None
```
and below have the featurizer instantiated explicitly as a `OneHotFeaturizer` if `None` is passed in via the constructor.

Reply:
I decided to go for a slightly different design where I just instantiate it with no parameters and then adjust the attributes after.

Is that acceptable?